test/rules/std_L048_test.py

```diff
@@ -0,0 +1,66 @@
+"""Tests the python routines within L048."""
```

Comment:
Is there any reason for these not to just be YML tests?

IMHO we should only use the python tests if there's something specific we wanna check that can't be covered by the YML tests  (e.g. two rules running at same time, or a different error message is returned than default when it should be).

Not seeing anything in below tests that couldn't be covered with a YML test case.

Yes you are checking that L048 is passing or failing, but as the YML test cases are only run for the specific rule, it's safe to assume if it fails it's because it's L048.

The text error messages are something we can't really check in YML but, as they are always the same message (though admittedly with a slightly different parameter), I'm not sure if they are worth adding the python test cases just for that and think sufficient to test for pass and fail. Maybe one Python test case to cover that (the `test__rules__std_L048_raised` one seems pretty comprehensive) and move the others to YML?